app/components/provider_interface/application_card_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="app-application-card">
+
+  <div class="app-application-card__primary">
+    <h3 id="applicant-name-<%= index %>" class="govuk-heading-m govuk-!-margin-bottom-3">
+      <%= link_to candidate_name, provider_interface_application_choice_path(application_choice), class: 'govuk-link' %>
+    </h3>
+    <p id="course-provider-name-<%= index %>" class="govuk-body govuk-!-font-size-16"><%= course_provider_name %></p>
+    <p id="course-name-and-code-<%= index %>" class="govuk-body govuk-!-margin-bottom-0"><%= course_name_and_code %></p>
+    <p id="accredited-body-<%= index %>" class="govuk-body govuk-caption-m govuk-!-font-size-16">Accredited body: <%= accrediting_provider.name if accrediting_provider %></p>
```

Comment:
True - think that all courses will have accredited bodies right? This isn't the case in the Dev test data generation though so I added this in so that the page would load (otherwise it calls a method on `nil`) Can amend to not output unless it is present though as this is cleaner.

Reply:
Not all courses have accredited bodies — it's only present where it's different from  `.provider`

Reply:
👍

Reply:
👍